### DISCUSSION TITLE
Is there any cli tool to generate JSON files and extract default values?

### DISCUSSION THREAD

**Discussion by Yuripetusko** [vinissimus/next-translate#792] [Q&A]
Is there anything similar to https://github.com/i18next/i18next-parser ?

**Comment by wuifdesign** [comment]
i created a package for this purpose, hope it works fine.
https://www.npmjs.com/package/next-translate-scanner

**Reply by Yuripetusko** [reply]
> i created a package for this purpose, hope it works fine. https://www.npmjs.com/package/next-translate-scanner

perfect, will try it out
